src/Extensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace StreamChat
+{
+    internal static class Extensions
+    {
+        internal static void ForEach<T>(this IEnumerable<T> items, Action<T> action)
+        {
+            if (items == null || action == null)
```

Comment:
(minor) probably action should not be allow null
if someone is doing
```
collection.ForEach(null);
```
it's most probably wrong and unintended usage

Reply:
You know what? I'll just remove that check because no one will ever do that anyway.